### PR TITLE
Adding a sitemap to improve SEO

### PR DESCRIPTION
**Pull request by svillafe**
## Why are you doing this?

Sitemaps are useful for search engines to know about the organization of a site. Search engine web crawlers like Googlebot read this file to more intelligently crawl your site.

Sitemap's protocol: https://www.sitemaps.org/protocol.html

This PR is inspired by @rtyley's PR in membership: https://github.com/guardian/membership-frontend/pull/1351

[**Trello Card**](https://trello.com/c/FiEiovzZ/974-add-sitemap)

## Changes

* Added `/sitemap.xml`

## Response

```xml

 <urlset xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
    <url>
      <loc>https://support.thegulocal.com/uk</loc>
       <xhtml:link rel="alternate" hreflang="en-us"    href="https://support.thegulocal.com/us/contribute"/>
       <xhtml:link rel="alternate" hreflang="en" href="https://support.thegulocal.com/uk"/>
       <priority>1.0</priority>
   </url>
   <url>
       <loc>
           https://support.thegulocal.com/us/contribute
       </loc>
       <xhtml:link rel="alternate" hreflang="en-us" href="https://support.thegulocal.com/us/contribute"/>
       <xhtml:link rel="alternate" hreflang="en" href="https://support.thegulocal.com/uk"/>
       <priority>1.0</priority>
   </url>
</urlset>
```

### CODE REVIEW

**Comment by JustinPinner** [comment]
Looks good 👍

File: app/wiring/AppComponents.scala [hunk 1]
```diff
@@ -35,10 +35,11 @@ trait AppComponents extends PlayComponents
   override lazy val router: Router = new _root_.router.Routes(
     httpErrorHandler,
     applicationController,
+    siteMapController,
     regularContributionsController,
     oneOffContributions,
     loginController,
-    testUsersContoller,
+    testUsersController,
```
Comment (rupertbates) [inline app/wiring/AppComponents.scala]: :1st_place_medal:

**Comment by rupertbates** [comment]
I think we could use the information [here](https://support.google.com/webmasters/answer/2620865?hl=en&ref_topic=2370587) to specify that /uk and /us/contribute are homepages for the uk and us respectively

**Comment by svillafe** [comment]
Good call @rupertbates. I updated the controller to handle the `hreflang` cases

**Comment by rupertbates** [comment]
Could you update the response in the PR description, just for reference?

**Comment by svillafe** [comment]
@rupertbates It is updated, did you try the hard refresh? sometimes github doesn't update on the fly the description

**Comment by rtyley** [comment]
> I think we could use the information here to specify that /uk and /us/contribute are homepages for the uk and us respectively

So far as I can see, you're respecting this bit of advice, which is the important thing:

```
You must create a separate url element for each URL. Each url element must include
a loc tag indicating the page URLs, and an xhtml:link rel="alternate" hreflang="XX" subelement
for every alternate version of the page, **including itself**. 
```

Once deployed, you'll want to check in https://www.google.com/webmasters/tools/home?hl=en&pli=1 that the sitemap is being interpreted correctly.

**Pull request merged**